examples/tornjak/values-ingress.yaml

```diff
@@ -0,0 +1,41 @@
+global:
+  spire:
+    jwtIssuer: https://oidc-discovery.$APP_SUBDOMAIN
```

Comment:
With all the rest of the ingress pr's in place and examples-your-values.yaml, I think the only thing left in this file that is needed in this file is the bits that enable various ingress's?